app/controllers/allocations_controller.rb

```diff
@@ -85,6 +85,12 @@ def create
     redirect_to summary_unallocated_path
   end
 
+  def history
+    offender = offender(nomis_offender_id_from_url)
```

Comment:
You might find this easier to pass @offender to the template, as this will make it easier to call `@offender.full_name` in the title and `@offender.nomis_no` in the backlink.